sunshine/config.cpp

```diff
@@ -573,6 +573,11 @@ int apply_flags(const char *line) {
 }
 
 void apply_config(std::unordered_map<std::string, std::string> &&vars) {
+  if(!fs::exists(stream.file_apps.c_str())) {
+      std::string sunshine_apps_default = SUNSHINE_DEFAULT_DIR "/" APPS_JSON;
+      fs::copy_file(sunshine_apps_default.c_str(), stream.file_apps.c_str());
```

Comment:
c++ filesystem function can take string directly.

Therefore, any function that starts with `fs::` can take strings directly.

No need to use `stream.file_apps.c_str()`, use `stream.file_apps` instead.